CONTRIBUTING.md

```diff
@@ -76,48 +76,50 @@ You should pick either the "personal" or "this computer" option.
 
 ## Making Changes
 
+FakeItEasy uses the git branching model known as [GitFlow](http://nvie.com/posts/a-successful-git-branching-model/). As such, all development must be performed on a ["feature branch"](https://martinfowler.com/bliki/FeatureBranch.html) created from the main development branch, which is called `develop`. To submit a change:
+
 1. [Fork](http://help.github.com/forking/) the  [FakeItEasy repository](https://github.com/FakeItEasy/FakeItEasy/) on GitHub
 1. Clone your fork locally
 1. Configure the upstream repo (`git remote add upstream git://github.com/FakeItEasy/FakeItEasy.git`)
-1. Create a local branch (`git checkout -b my-branch`)
+1. Create a local branch (`git checkout -b my-branch develop`)
 1. Work on your feature
 1. Rebase if required (see below)
 1. Run code analysis on the solution to ensure you have not introduced any violations
-1. Ensure the build succeeds (see ['How to build'](https://github.com/FakeItEasy/FakeItEasy/blob/master/how_to_build.md "How to build"))
+1. Ensure the build succeeds (see ['How to build'](how_to_build.md "How to build"))
 1. Push the branch up to GitHub (`git push origin my-branch`)
 1. Send a [pull request](https://help.github.com/articles/using-pull-requests) on GitHub
 
-You should **never** work on a clone of master and you should **never** send a pull request from master - always from a branch. The reasons for this are detailed below.
+You should **never** work on a clone of develop and you should **never** send a pull request from develop - always from a branch. The reasons for this are detailed below.

```

Comment:
This doesn't seem very clear to me... you clone a repo, not a branch. It should probably be rephrased (maybe something like "you should never work directly on the `develop` branch")